components/automate-gateway/api/iam/v2/teams.proto

```diff
@@ -77,7 +245,45 @@ service Teams {
     option (chef.automate.api.iam.policy).resource = "iam:teams:{id}";
     option (chef.automate.api.iam.policy).action = "iam:teamUsers:create";
   };
+
+  /*
+  Removes local team membership
+
+  Removes a list of users from a local team. Users are removed by their membership_id.
```

Comment:
I think it's worth adding another line that a user's membership ID can be found as part of the user details returned from GetUser (or something)